mirror/src/main/java/com/bakdata/quick/mirror/MirrorApplication.java

```diff
@@ -80,17 +80,26 @@
     @Option(names = "--retention-time", description = "Retention time defined in ISO_8601")
     private Duration retentionTime;
 
+    @Nullable
+    @Option(names = "--range", description = "The field which the Mirror builds its range index on")
```

Comment:
```suggestion
    @Option(names = "--rangeField", description = "The field which the Mirror builds its range index on")
```

I think we call it `rangeField` everywhere else, so why not here? This would also require changing the argument in the manager when adding the argument